scripts/validate_docstrings.py

```diff
@@ -585,6 +586,20 @@ def validate_one(func_name):
 
     errs = []
     wrns = []
+
+    if len(doc.raw_doc) == 0:
+        errs.append(error('GL08'))
+        return {
```

Comment:
Do we need to explicitly return here? Would be more inline with rest of the code if we allow it to pass over the remaining conditions (which would assumedly be all False anyway) and return at the end of the function

Reply:
The rest of the condition will not be False, if we don't do early return the next two conditions would evaluate to true

doc.start_blank_lines
doc.end_blank_lines

it's possible to put the code underneath into all-encompassing else statement, but the early return just seems cleaner.